core/src/main/java/org/aerogear/mobile/core/metrics/MetricsService.java

```diff
@@ -79,6 +77,8 @@ public void sendAppAndDeviceMetrics(final Callback callback) {
     /**
      * Send metrics
      *
+     * WARN: Not intended by direct usage outside of the SDK
+     *
```

Comment:
@paolobueno Let's create an annotation and add it to our linter so that this warning is enforceable at compile time

Reply:
@paolobueno It doesn't require an annotation processor, we just have to code a rule for it in the lint rules which are in the codequality directory and rebuild the checks.jar file  I can help you with it because it is areally fun to do if you like source parsing and processing.